tutorials/vph/tutorial_digitaltwins_on_fhir.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<p>TODO? or you meant "Step"?</p><p>Also:</p><p> </p><ul><li>Let's find <strong>out </strong>which <s>workflow / wolkflow tool</s> <strong>workflows or workflow tools</strong> <s>generate </s> <strong>generated </strong>this observation</li><li>Let's find <strong>out </strong><s>the observation related to which dataset</s> <strong>which datasets are related to this observation</strong></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ABI-CTT-Group/digitaltwins-api/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>